pkg/kratos/service.go

```diff
@@ -316,6 +393,42 @@ func (s *Service) ParseLoginFlowMethodBody(r *http.Request) (*kClient.UpdateLogi
 	return &ret, nil
 }
 
+func (s *Service) ParseRecoveryFlowMethodBody(r *http.Request) (*kClient.UpdateRecoveryFlowBody, error) {
+	body := new(kClient.UpdateRecoveryFlowWithCodeMethod)
+
+	err := parseBody(r.Body, &body)
+
+	if err != nil {
+		return nil, err
+	}
+	ret := kClient.UpdateRecoveryFlowWithCodeMethodAsUpdateRecoveryFlowBody(
+		body,
+	)
+
+	email := "test@example.com"
```

Comment:
issue: I understand that this is the only existing user right now, but can we avoid hardcoding this and take this value from the UI form ? So this would work also for other account when there are some